patches.json

```diff
@@ -225,5 +228,108 @@
+  },
+  "MDVA-28409": {
+    "magento/magento2-base": {
+      "Fixes the issue where the 'sales_clean_quotes' cron job fails with out-of-memory error when the number of expired quotes in the database is huge.": {
+        ">=2.3.4 <2.3.6 || 2.4.0": {
+          "file": "os/MDVA-28409__cleaning_old_quotes_from_sales_clean_quotes_table__2.3.4.patch"
+        },
+        ">=2.3.0 <2.3.4": {
+          "file": "os/MDVA-30593__cleaning_old_quotes_from_sales_clean_quotes_table__2.3.3.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30107": {
+    "magento/magento2-base": {
+      "Fixes the issue where store switcher doesn't work as expected if different base URLs are used for store views.": {
+        ">=2.3.0 <2.3.5": {
+          "file": "os/MDVA-30107__fixes_store_switcher_for_different_base_url__2.3.3.patch"
+        },
+        ">=2.3.5 <2.3.6": {
+          "file": "os/MDVA-30623__fixes_store_switcher_for_different_base_url__2.3.5-p2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-28763": {
+    "magento/magento2-base": {
+      "Fixes the issue where product image is getting duplicated after updating product information using REST API more than once.": {
+        ">=2.3.2 <2.3.4": {
+          "file": "os/MDVA-28763__fixes_product_image_is_getting_duplicate_using_REST_API__2.3.3.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30284": {
+    "magento/magento2-base": {
+      "Fixes the issue where Catalog Search indexer fails due to the following Elasticsearch error: `limit of total fields in index has been exceeded`.": {
+        ">=2.3.0 <2.4.2": {
+          "file": "os/MDVA-30284__fixes_catalog_search_indexer_fail_due_to_elasticsearch_error__2.3.5-p2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-29042": {
+    "magento/magento2-b2b-base": {
+      "Fixes the issue where Catalog permissions were changed to `Allow` automatically after new product was added to the shared catalog.": {
+        ">=1.1.3 <=1.1.4-p2": {
+          "file": "commerce/MDVA-29042__fixes_catalog_permission_changing_to_allow__2.3.4.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30428": {
+    "magento/magento2-base": {
+      "Fixes the issue where customers cannot add a product to wishlist if this product is assigned to a custom inventory source.": {
+        ">=2.3.5 <2.4.2": {
+          "file": "os/MDVA-30428__fixes_add_a_product_with_custom_inventory_to_wishlist__2.3.5-p1.patch"
+        }
+      }
+    },
+    "magento/inventory-metapackage": {
+      "Fixes the issue where customers cannot add a product to wishlist if this product is assigned to a custom inventory source.": {
+        ">=1.1.5 <1.2.2": {
+          "file": "os/MDVA-30428__fixes_add_a_product_with_custom_inventory_to_wishlist__1.1.5.patch"
+        }
+      }
+    }
+  },
+  "MDVA-28661": {
+    "magento/magento2-b2b-base": {
+      "Fixes the issue where an error is thrown in the `Company Users` company account section after company admin is changed.": {
+        ">=1.1.0 <1.2.2": {
```

Comment:
This patch is not applicable to 1.1.5-p2. Please review version contraints.
Suggested: `>=1.1.0 <1.1.5-p2 || >=1.2.0 <1.2.2`

Reply:
Fixed

Reply:
@ilima-ebay  @Rykh  MQP build shows that the patch is applicable to 1.1.5-p2
I checked manually and it's applicable
We need to check the http://web2.sparta.corp.magento.com/dev/support/tools/patchchk/ tool if it works correctly with security releases

Reply:
@viktym @Rykh yes, I checked the patch checker tool and the 2.3.5-p2 installation was missing b2b that's why it was not applicable. I fixed it and it now shows applicable. @Rykh  i think you can change it back to `>=1.1.0 <1.2.2`. Sorry about that